### PR TITLE
Add section with description of atmos_model_nml.

### PR DESCRIPTION
**Pull request by JulieSchramm**
## Description

Add a description of the variables in atmos_model_nml to the Users Guide.

## Testing

html and latex builds succeed

## Dependencies

None

### CODE REVIEW

File: doc/UsersGuide/source/InputNML.inc [hunk 1]
```diff
@@ -224,4 +230,92 @@ A sample subset of this namelist is shown below:
...
+   * - fhmax
+     - The maximal forecast time for output.
+     - real
+     - 384.0
+   * - fhmaxhf
+     - The maximal forecast hour for high frequency output.
+     - real
+     - 120.0
+   * - fhout
+     - Output frequency during forecast time from 0 to ``fhmax``, or from ``fhmaxhf`` to ``fhmaxhf`` if ``fhmaxf>0``.
```
Comment (ligiabernardet) [inline doc/UsersGuide/source/InputNML.inc]: I am not understanding this definition. Does this make sense: from fhmaxhf to fhmaxhf???
Should it be (...) or from fhmaxhf to fhmax if fhmaxhf>0?

**Comment by junwang-noaa** [comment]
It should be   fhmaxhf to fhmax

**Review by ligiabernardet — CHANGES_REQUESTED** [review]
Tks for fixing nmelist description.
In introduction and in Chapter 5, https://stochastic-physics.readthedocs.io/en/release-ufs-v1.1.0/ is a broken link. Should be https://stochastic-physics.readthedocs.io/en/ufs-v1.1.0/.
Please review all links. They all should work EXCEPT for those that refer to the ufs-weather-model tag for this release, which has not been created yet.

**Comment by JulieSchramm** [comment]
This doesn't work:

https://dtcenter.org/GMTB/v4.1.0/sci_doc/CCPPsuite_nml_desp.html

**Comment by JulieSchramm** [comment]
Maybe this:

https://dtcenter.ucar.edu/GMTB/v4.1.0/sci_doc/CCPPsuite_nml_desp.html

**Comment by ligiabernardet** [comment]
@Laurie Carson <carson@ucar.edu> My understanding is that we should not
give the users any link to dtcenter.ucar.edu, only dtcenter.org.
The main link for the SciDoc (https://dtcenter.org/GMTB/v4.1.0/sci_doc/)
works but redirected to ucar.edu.
The links underneath (e.g., to namelist description) are only working on
ucar.edu.
Can you please advise?

**Review by ligiabernardet — APPROVED** [review]

**Review by climbfuji — APPROVED** [review]
Looks good. @junwang-noaa @DusanJovic-NOAA this is the last missing documentation update to merge into release/public-v1 of the ufs-weather-model before it can be tagged as ufs-v1.1.0.

**Pull request merged**